pkg/network/common/dns.go

```diff
@@ -27,6 +28,9 @@ type dnsValue struct {
 }
 
 type DNS struct {
+	// Runs shell commands
+	execer kexec.Interface
```

Comment:
Again part of the conflict, not quite sure if there is any advantage of replacing exec for kexec